js/modules/swiperHelpers.js

```diff
@@ -0,0 +1,35 @@
+export function setSlidesState(slider, className){
+	const activeIndexes = getActiveIndexes(slider, slider.realIndex);
+	const slideCollection = slider.slides;
+	
+	slideCollection.forEach(function(elem, index) {
+		if(activeIndexes.includes(index)){
+			elem.classList.remove(className);
+		}
+		else{
+			elem.classList.add(className);
+		}
+	});
+}
+
+export function getActiveIndexes(slider, realIndex){
+	let activeIndexes = [];
+	let activeIndex = realIndex;
+	if(realIndex == 0){
+		for(let i = 0; i < slider.params.slidesPerView; i++){
+			activeIndexes.push(i);
+		}
+	}
+	if(realIndex == slider.slides.length - slider.params.slidesPerView){
+		for(let i = realIndex; i < slider.slides.length; i++){
+			activeIndexes.push(i);
+		}
+	}
+	else {
+		const quantityOfActiveIndexes = slider.params.slidesPerView - 2;
```

Comment:
1)2 - is a magic number, when I read the code I don't understand, what it means and why it is here.
2)looks like you not only get active slides but also have some additional logic, then rename function so that anybody can understand what it does without reading a code